src/components/special/_rounded-diagonal.less

```diff
@@ -0,0 +1,19 @@
+.rounded-diagonal-btn {
+  background: #7c26c2;
```

Comment:
Instead of giving it a background color, use one of the colors we have in the example (for example purple-btn). This will help the person using this button to change the color by using whatever colored classes we have.

Reply:
I think that's special category button which is why I used css file for background-color.

Reply:
It lies under special category, instead of using 'purple-btn' from js, I prefer to use the same color from **sbuttons.less**.

Reply:
If I add second class from js file then it won't show the period in `class snippet`.

![image](https://user-images.githubusercontent.com/35252877/94910652-a3034880-04c2-11eb-8c0b-a49edf4bac74.png)

Reply:
Added !